packages/native/test/lib/ElementAssertion.test.tsx

```diff
@@ -271,4 +271,64 @@ describe("[Unit] ElementAssertion.test.ts", () => {
       });
     });
   });
+
+  describe (".toContainElement", () => {
+    const element = render(
+      <View testID="grandParentId">
+        <View testID="parentId">
+          <View testID="childId" />
+        </View>
+        <Text testID="textId" />
+      </View>,
```

Comment:
Can we use `<Text>` elements so that we can avoid the testIDs?

Reply:
Im trying to do this, but im not getting good results, would you like to pair?